src/renderers/shared/fiber/__tests__/ReactIncremental-test.js

```diff
@@ -98,21 +102,21 @@ describe('ReactIncremental', () => {
       );
     }
 
-    ReactNoop.render(<Foo text="foo" />);
+    ReactNoop.render(<Foo text="foo" />, () => ops.push('renderCallbackCalled'));
     ReactNoop.flush();
 
-    expect(ops).toEqual(['Foo', 'Header', 'Content', 'Footer']);
+    expect(ops).toEqual(['Foo', 'Header', 'Content', 'Footer', 'renderCallbackCalled']);
 
     ops = [];
 
-    ReactNoop.render(<Foo text="bar" />);
+    ReactNoop.render(<Foo text="bar" />, () => ops.push('renderCallbackCalled'));
     ReactNoop.flush();
 
     // TODO: Test bail out of host components. This is currently unobservable.
 
     // Since this is an update, it should bail out and reuse the work from
     // Header and Content.
-    expect(ops).toEqual(['Foo', 'Content']);
+    expect(ops).toEqual(['Foo', 'Content', 'renderCallbackCalled']);
 
   });
```

Comment:
Should we also have a test verifying that if you queue two callbacks without flushing, and then flush once, both get called?